crt/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ include(CTest)
 
 option(AWS_USE_LIBCRYPTO_TO_SUPPORT_ED25519_EVERYWHERE "Set this if you want to use libcrypto to support ed25519 on Window/Apple" ON)
 
+# Use minimal debug info
```

Comment:
Can you please add more details on why we are doing it i.e to reduce artifact size?